tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+  "ts-node": {
+    "transpileOnly": true
+  },
+  "compilerOptions": {
+    "noEmit": true,
+    "module": "CommonJS",
+    "target": "ES2019",
+    "moduleResolution": "node",
+    "lib": ["esnext", "es2018"],
+    "allowSyntheticDefaultImports": true,
+    "skipLibCheck": true,
+    "resolveJsonModule": true,
+    "strictNullChecks": true,
+    "experimentalDecorators": true,
```

Comment:
we dont use it, so no need for it
```suggestion
```

Reply:
tsconfig changes doesn't really matters. It runs only once on CI.
I will remove with next update within pr.